index.js

```diff
@@ -1,6 +1,15 @@
-exports.Metrics = require('./metrics');
-exports.StringReporter = require('./reporters/string-reporter');
-exports.ConsoleReporter = require('./reporters/console-reporter');
-exports.GraphiteReporter = require('./reporters/graphite-reporter');
-exports.DataDogReporter = require('./reporters/datadog-reporter');
-exports.InMemoryReporter = require('./reporters/in-memory-reporter');
+const { ConsoleReporter } = require('./src/reporters/console-reporter');
```

Comment:
you can add index file in the `src` and add "main" in package json. then all the code will be under the `src`